Dockerfile.filedatabase

```diff
@@ -9,5 +9,5 @@ LABEL org.label-schema.schema-version="1.0"
 LABEL org.label-schema.build-date=$BUILD_DATE
 LABEL org.label-schema.vcs-url="https://github.com/EGA-archive/LocalEGA"
 LABEL org.label-schema.vcs-ref=$SOURCE_COMMIT
-COPY --from=builder ega-data-api-filedatabase/target/ega-data-api-res-0.0.1-SNAPSHOT.jar .
+COPY --from=builder ega-data-api-filedatabase/target/ega-data-api-fileserver-0.0.1-SNAPSHOT.jar .
```

Comment:
Are you sure this is not ega-data-api-filedatabase?

Reply:
Of course.